app/views/instruments/_instrument_manage_fields.html.haml

```diff
@@ -4,11 +4,18 @@
 
 = f.input :problems_resolvable_by_user, as: :boolean, hint: false
 
-= f.input :pricing_mode
+- daily_booking = @product.daily_booking?
 
-= f.input :reserve_interval, value_method: :to_i, label: "Reservation Interval"
-= f.input :min_reserve_mins, value_method: :to_i
-= f.input :max_reserve_mins, default_value: "None"
+- if SettingsHelper.feature_on?(:show_daily_rate_option) || !daily_booking
+  = f.input :pricing_mode
+
+- if daily_booking
+  = f.input :min_reserve_days, value_method: :to_i
+  = f.input :max_reserve_days, value_method: :to_i
+- else
+  = f.input :reserve_interval, value_method: :to_i, label: "Reservation Interval"
+  = f.input :min_reserve_mins, value_method: :to_i
+  = f.input :max_reserve_mins, default_value: "None"
```

Comment:
If I'm reading this right, I think you could simplify a bit:
```suggestion
- if @product.daily_booking?
  - if SettingsHelper.feature_on?(:show_daily_rate_option)
    = f.input :pricing_mode
  = f.input :min_reserve_days, value_method: :to_i
  = f.input :max_reserve_days, value_method: :to_i
- else
  = f.input :pricing_mode
  = f.input :reserve_interval, value_method: :to_i, label: "Reservation Interval"
  = f.input :min_reserve_mins, value_method: :to_i
  = f.input :max_reserve_mins, default_value: "None"
```

Reply:
We agreed only setting the value was going to be behind a Feature Flag, right? In that case, I think the `if` in line 13 is unnecessary. 

That said, I took a look at the code and this should be right:
```
# No if condition here
= f.input :pricing_mode

- if daily_booking
  = f.input :min_reserve_days, value_method: :to_i
  = f.input :max_reserve_days, value_method: :to_i
- else
  = f.input :reserve_interval, value_method: :to_i, label: "Reservation Interval"
  = f.input :min_reserve_mins, value_method: :to_i
  = f.input :max_reserve_mins, default_value: "None"
```

WDYT?